config/routes.rb

```diff
@@ -1,3 +1,21 @@
+#Draw routes
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  #Set root path
```

Comment:
Same, add space after `#` and `.` at end of all comments. I'm pretty sure iomt-rubocop would've caught these.